### DISCUSSION TITLE
drive storage limit

### DISCUSSION THREAD

**Discussion by IndrajeethY** [deta/cloud-docs#325] [Help] [answered]
how do i check database storage limit using python?

**Comment by jnsougata** [comment, marked as answer]
Currently there's no way to check it either from UI or from SDK/HTTP Endpoint.

**Comment by abdelhai** [comment]
hi, your title says "drive" and your question says "database", these are two different things.
drive has a limit of 10gb per account. there's no way to check how much you have used at the moment.
base (the database) doesn't have a limit and there's no way to how much you have used.

**Reply by IndrajeethY** [reply]
oh sorry. i meant database and thanks for reply
